src/content/contributing/security/_index.en.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Security Reporting"
+weight: 80
+---
+
+Submariner welcomes and appreciates responsible disclosure of security vulnerabilities.
+
+If you know of a security issue with Submariner, please report it to submariner-security@googlegroups.com.
```

Comment:
Can private groups be created on Google Groups? (I should look this up...)

Reply:
It seems like it. Looking at https://groups.google.com/g/submariner-dev/about, I see settings like "Anyone on the web
can view conversations" and I imagine we can configure that to group members only.

Ping @mangelajo to try to set this up.

Reply:
Okay @skitt, I added secrutiy.txt files in a second commit.